samples/static/country/id.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
in long term, it makes sense to fetch this from a database correct?

Reply:
@shan-96 for us it doesn't matter, it could be rest api as well, or for non-be developers it could be useful just to use static json file instead of hardcoding values on FE side.